composites/HelpCenter/views/VideoTutorial.js

```diff
@@ -102,10 +106,14 @@ VideoDescriptionItem.defaultProps = {
 export default function VideoTutorial( props ) {
 	return (
 		<VideoTutorialContainer className={ `${ props.className }__container` }>
-			<VideoContainer className={ `${ props.className }__video-container` }>
+			<VideoContainer
+				className={ `${ props.className }__video-container` }
+				hasAdvertisements={ props.paragraphs.length > 0 }
```

Comment:
I would change the naming. You want to align the video if there is no paragraph. Advertisements is more like an implementation details. Can you change it to: hasParagraphs or something like that.

Reply:
👍